src/styles/cards/card.scss

```diff
@@ -0,0 +1,89 @@
+.card {
+  width: $card-width;
+  border-radius: 5px;
+  border: 1px solid $border-color-card;
+  justify-content: center;
+  display: block;
+  flex-direction: column;
+  padding: 16px;
+  transition: transform 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    padding: 16px 4px 30px;
+    display: flex;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: $line-height;
+    font-weight: 500;
+    color: $color-title;
+    margin-bottom: -6px;
+  }
+
+  &__title:hover {
+    color: #34568b; // Правило для ховера на заголовок отдельно
+  }
+
+  &__subtitle {
+    font-size: 10px;
+    line-height: 14px;
+    color: $color-subtitle;
+  }
+
+  &__review {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    font-size: 10px;
+    margin-bottom: 14px;
+  }
+
+  &__price {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    &-label {
+      font-size: 12px;
+      line-height: $line-height;
+      color: $color-subtitle;
+    }
+
+    &-value {
+      font-size: 16px;
+      font-weight: bold;
+      line-height: $line-height;
+    }
+  }
+
+  &__button {
+    font-family: Roboto, sans-serif;
+    display: block;
+    width: 100%;
+    padding-block: 11px;
+    border-radius: 5px;
+    text-transform: uppercase;
+    text-decoration: none;
+    border: 1 solid $button-color;
+
+    font-size: 14px;
+    font-weight: 700;
+    color: white;
+    background-color: $button-color;
+    line-height: 16px;
+    text-align: center;
+  }
+
+  &__button:hover {
+    background-color: #fff;
+    color: $button-color;
+  }
```

Comment:
On hover, the button text color and background color are switching places. This could lead to accessibility issues as the contrast may not be sufficient for all users. Consider choosing a hover state that maintains high contrast for better accessibility.